lib/crop.js

```diff
@@ -85,7 +85,12 @@ Crop.prototype.exec = function () {
     }
     context.stream(streamOutput)
   } else if (imageFileType.ext === 'gif') {
-    var crop = this.crop.x1 + ',' + this.crop.y1 + '+' + this.crop.w + 'x' + this.crop.h
+    // this is the rounding grpahicsmagick uses for geometry
+    var roundedX1 = Math.ceil(this.crop.x1 - 0.5)
```

Comment:
Is this different from `Math.round(x)`?

Reply:
Yes, without it the test with a crop xx.5 will fail as it is off by one pixel compared to graphicsmagick, which uses ceil(number - 0.5) internally for coords and floor(number + 0.5) for dimensions

Reply:
You just totally pwned me Ryan. 

I am very happy to learn something new and impressed with how you took the time to see how GM does things.